src/OrbitGl/CoreMathTest.cpp

```diff
@@ -8,19 +8,19 @@
 
 namespace orbit_gl {
 
-const ClosedInterval kSmallClosedInterval{0, 3};
+const ClosedInterval<int> kSmallClosedInterval{0, 3};
```

Comment:
Your usage of ClosedInterval is float-only (so far) but the tests are all with `int`. What about adding a `TYPED_TEST_SUITE`/`TYPED_TEST`?